assets/js/components/Loadpoints/SettingsModal.vue

```diff
@@ -282,10 +280,11 @@ export default defineComponent({
 		changePhasesConfigured() {
 			this.$emit("phasesconfigured-updated", this.selectedPhases);
 		},
-		currentOption(value: number, isDefault: boolean) {
-			let name = `${this.fmtNumber(value, value <= 1 ? undefined : 0)} A`;
+		currentOption(value: number, isDefault: boolean, isMax: boolean) {
```

Comment:
Vorschlag: lass uns hier direkt die Phasenanzahl übergeben. Die Multiplikation mit V braucht dann ja auch nur hier gemacht werden.

```suggestion
		currentOption(value: number, isDefault: boolean, phases: number) {
```